src/redis4net/Appender/RedisAppender.cs

```diff
@@ -12,6 +13,16 @@ public class RedisAppender : AppenderSkeleton
 		public string RemoteAddress { get; set; }
 		public int RemotePort { get; set; }
 		public string ListName { get; set; }
+		public bool AbortOnConnectFail { get; set; }
+		public int ConnectRetry { get; set; }
+		public int ConnectTimeOut {get; set; }
+
+		public RedisAppender()
+		{
+			AbortOnConnectFail = false;
```

Comment:
Not always you can override them if you want in the config. The values that are being used are the values that where already in the code, so there is not an other behavior after updating on that specific part.